### PR TITLE
Add target folder to load path

### PR DESCRIPTION
**Pull request by ryanmelt**
This will allow requires like: require "TARGET/lib/my_file.rb" to work inside of microservices.

### CODE REVIEW

File: openc3/lib/openc3/io/json_api.rb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+#
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+
+require 'openc3/io/json_api_object'
+
+module OpenC3
+  class JsonApi
+    # Create a JsonApiObject connection to the API server
+    def initialize(microservice_name: 'CFDP', prefix: '/cfdp', schema: 'http', hostname: nil, port: 2905, timeout: 5.0, url: nil, scope: $openc3_scope)
```
Comment (jmthomas) [inline openc3/lib/openc3/io/json_api.rb:24]: Did you mean to check this in? What are these defaults?

**Review by jmthomas — COMMENTED** [review]
json_api.rb?

File: openc3/lib/openc3/io/json_api.rb (hunk 1 above)
Comment (ryanmelt) [reply openc3/lib/openc3/io/json_api.rb:24]: I did sneak that file in.  But you're write I should cleanup the defaults

**Review by jmthomas — APPROVED** [review]

File: openc3/lib/openc3/io/json_api.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+# GNU Affero General Public License for more details.
+#
+# This file may also be used under the terms of a commercial license
+# if purchased from OpenC3, Inc.
+
+require 'openc3/io/json_api_object'
+require 'openc3/utilities/authentication'
+
+module OpenC3
+  class JsonApi
```
Comment (jmthomas) [inline openc3/lib/openc3/io/json_api.rb:23]: How / where is this used?

Comment (ryanmelt) [reply openc3/lib/openc3/io/json_api.rb:23]: Its used by the new CFDP code, and should be used in a future API generator

**Pull request merged**